spec/features/user_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'capybara/rspec' 
+
+feature 'Admin can see a list of users' do
+  scenario 'clicks user list' do
+    visit root_path
+    click_link 'User List'
+    expect(page).to have_content('Users')
+  end
+end
+
+feature 'Admin creates a new user' do
+  scenario 'clicks add user button' do
+    visit users_path
+    click_link 'Add User'
+    expect(page).to have_content('New User')   
+  end
+  scenario 'with attributes for Admin' do
+    visit new_user_path
+    fill_in 'Full Name', with: 'Test'
+    fill_in 'Email', with: 'email@email.com'
+    check 'Is Admin'
+    click_button 'Submit'
+    expect(page).to have_content('Test')
+    expect(page).to have_content('email@email.com')
+    expect(page).to have_content('Admin')
+  end
+  scenario 'with attributes for Team Leader' do
+    visit new_user_path
+    fill_in 'Full Name', with: 'Test1'
+    fill_in 'Email', with: 'email1@email.com'
+    fill_in 'Team Leader', with: '1'
+    click_button 'Submit'
+    expect(page).to have_content('Test1')
+    expect(page).to have_content('email1@email.com')
+    expect(page).to have_content('Team Leader')
+  end
+  scenario 'with attributes for Team Member' do
+    visit new_user_path
+    fill_in 'Full Name', with: 'Test2'
+    fill_in 'Email', with: 'email2@email.com'
+    click_button 'Submit'
+    expect(page).to have_content('Test2')
+    expect(page).to have_content('email2@email.com')
+    expect(page).to have_content('Team Member')
+  end
+end
+
+    
```

Comment:
Please remove the extra lines at the end of this file, there should only be 1 blank line at the end of the file by default. You might check your editor settings as well; you have a lot of files in this PR that are removing the blank lines at the ends of files, and that's often a "trim whitespace" option that will edit your files by default on save. Our team's practice is to strip whitespace at the ends of line, and add/preserve a single blank line at the end of code files. If not everyone has the same settings on their editor, you could be modifying each others' changes without even realizing it, which adds visual noise to git diffs.

Reply:
Okay. I went through and checked files with extra lines and removed them. I also enabled trim trailing whitespace in vs code.

Reply:
The consistency across files is good! But I actually meant that all files should have a single extra blank line at the end of them. I'm sorry, I didn't communicate that very well 😅In VSCode, this setting is called `files.insertFinalNewline`.